recipe/fix_empty_cmake_variable.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index b230b9e6d..446a6eed3 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -127,6 +127,7 @@ set_target_properties(yaml-cpp PROPERTIES
+   PROJECT_LABEL "yaml-cpp ${yaml-cpp-label-postfix}"
+   DEBUG_POSTFIX "${CMAKE_DEBUG_POSTFIX}")
+ 
++set(EXPORT_TARGETS yaml-cpp)
```

Comment:
What do you think about including the [full patch](https://github.com/jbeder/yaml-cpp/pull/1077/files#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20a), since it was merged upstream?

Reply:
I initially tried that, but there were a lot of conflicitng changes that I was not fully understanding, so I preferred to go for the simplest fix that I was sure was not creating regressions.